tests/e2e/tutorials/tutorialBase.js

```diff
@@ -35,12 +41,18 @@ class TutorialBase {
   }
 
   async start() {
-    await this.beforeScript();
-    await this.goTo();
+    try {
+      await this.beforeScript();
+      await this.goTo();
 
-    const needsRegister = await this.registerIfNeeded();
-    if (!needsRegister) {
-      await this.login();
+      const needsRegister = await this.registerIfNeeded();
+      if (!needsRegister) {
+        await this.login();
+      }
+    }
+    catch(err) {
+      console.error("Error starting", err);
+      throw(err);
```

Comment:
If you throw an exception, cannot you write the stack when caught??

Reply:
So far I didn't find it necessary.